pkg/internal/checkup/checkup.go

```diff
@@ -83,18 +83,29 @@ func New(client kubeVirtVMIClient, namespace string, checkupConfig config.Config
 	}
 }
 
-func (c *Checkup) Setup(ctx context.Context) error {
+func (c *Checkup) Setup(ctx context.Context) (setupErr error) {
 	const errMessagePrefix = "setup"
 	var err error
 
 	if err = c.createVMI(ctx); err != nil {
 		return fmt.Errorf("%s: %w", errMessagePrefix, err)
 	}
+	defer func() {
+		if setupErr != nil {
```

Comment:
I'm sure I'm missing something here, but it looks like `setupErr` never gets any value

Reply:
ah I see now. thanks!